plugins/gitea-comment/content.yaml

```diff
@@ -0,0 +1,73 @@
+title: Gitea Comment
+author: DefinitelyADev
+tags:
+  - infrastructure
+  - gitea
+  - drone
+  - comment
+  - buildstatus
+repo: https://github.com/DefinitelyADev/gitea-comment
```

Comment:
hey, do you mind adding an image - you could even use the existing gitea image - `logo: gitea.svg`